integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSource.scala

```diff
@@ -108,6 +110,10 @@ class CarbonSource extends CreatableRelationProvider
 
     val dbName: String = parameters.getOrElse("dbName", CarbonCommonConstants.DATABASE_DEFAULT_NAME)
     val tableName: String = parameters.getOrElse("tableName", "default_table")
+    if(org.apache.commons.lang.StringUtils.isBlank(tableName) ||
+       org.apache.commons.lang.StringUtils.isWhitespace(tableName)) {
+      throw new MalformedCarbonCommandException("INVALID TABLE NAME")
```

Comment:
@jackylk  i think emptyTableName will make less sense because it is not gurranted that it will be empty